ArduCopter/mode_planckrtb.cpp

```diff
@@ -22,8 +22,12 @@ bool Copter::ModePlanckRTB::init(bool ignore_checks){
 }
 
 void Copter::ModePlanckRTB::run(){
-    if(!_is_landing)
-    {
+    //Check for a case where we requested the landing but our request was rejected
+    if(_is_landing && copter.planck_interface.was_last_request_rejected()) {
```

Comment:
Is there a case where it would check for the wrong ack ? 
It checks for the last cmd sent assuming the last one is `PLANCK_CMD_REQ_LAND`

Reply:
It checks for the correct ack here: https://github.com/planckaero/ardupilot/pull/8/files#diff-119e514fc86ac541e0f5f0ddef6f50b5R21

Reply:
Yes, it checks the correct ack the the **last sent cmd**,
Here the assumption is that the last sent cmd is `PLANCK_CMD_REQ_LAND`

Reply:
yes, why would it be anything else?

Reply:
That's what I was wondering, if there is a case where there is another cmd sent right after the `PLANCK_CMD_REQ_LAND`